src/main/java/com/owncloud/android/lib/resources/status/CapabilityBooleanType.java

```diff
@@ -24,54 +24,55 @@
  */
 package com.owncloud.android.lib.resources.status;
 
+import androidx.annotation.NonNull;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 
 /**
  * Enum for Boolean Type in OCCapability parameters, with values:
  * -1 - Unknown
- *  0 - False
- *  1 - True
+ * 0 - False
+ * 1 - True
  */
 @AllArgsConstructor
 public enum CapabilityBooleanType {
-    UNKNOWN (-1),
-    FALSE (0),
-    TRUE (1);
+    UNKNOWN(-1),
+    FALSE(0),
+    TRUE(1);
 
-    @Getter private int value;
+    @Getter
+    private int value;
 
-    public static CapabilityBooleanType fromValue(int value)
-    {
-        switch (value)
-        {
+    public static @NonNull
+    CapabilityBooleanType fromValue(int value) {
+        switch (value) {
             case -1:
                 return UNKNOWN;
             case 0:
                 return FALSE;
             case 1:
                 return TRUE;
         }
-        return null;
+        return UNKNOWN;
     }
 
-    public static CapabilityBooleanType fromBooleanValue(boolean boolValue){
-        if (boolValue){
+    public static CapabilityBooleanType fromBooleanValue(boolean boolValue) {
+        if (boolValue) {
             return TRUE;
         } else {
             return FALSE;
         }
     }
 
-    public boolean isUnknown(){
+    public boolean isUnknown() {
         return getValue() == -1;
```

Comment:
same here `UNKNOWN.getValue()` or maybe `UNKNOWN.value`, same for the other checks